MozillaFoundation/foundation.mozilla.org
# Remove wagtail utility routes for authentication

# Description

This PR removes the base wagtail utility routes from unauthenticated user access, while preserving custom url patterns needed by the app.

# To test

Link to `/_util/login` (should 404): https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/_util/login/
Link to custom url serve pattern (should both work):
- https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/privacynotincluded/categories/health-exercise/
- https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/privacynotincluded/categories/health%20&%20exercise/

Link to sample test page: https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/privacynotincluded/categories/health-exercise/
Related PRs/issues: #12062

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-993)
